Automattic/wp-calypso
# Updates the Blaze App header and theme

We are working on updating the theme of the Blaze App for Jetpack. This PR updates the header and the theme for the Ready to Promote page.

![Screenshot 2023-06-14 at 17 45 29](https://github.com/Automattic/wp-calypso/assets/1258162/116150b3-907c-43bd-93a7-26a9d0cd8d6e)

## Proposed Changes

* New Jetpack Blaze header
* Updates styles inside the Blaze app
* Fixes navigation to the details page

## Testing Instructions
You need to have [this Jetpack branch](https://github.com/Automattic/jetpack/pull/30103) working locally.

* Check out this branch
* On a console, go to `apps/blaze-dashboard`
* Execute: `BLAZE_DASHBOARD_PACKAGE_PATH=/path/to/jetpack/projects/packages/blaze yarn dev`. 
You will need to change the path to the correct jetpack folder
* On another console, go to the jetpack folder and compile the blaze package: `pnpm jetpack build packages/blaze`
* Open the jurassic tunel to your local jetpack and go to the wp-admin page
* Click on Tools->Advertising (if the option is not shown, you need to activate the blaze gate. There is an explanation on the jetpack PR).
* Verify that the new Jetpack Blaze header is shown (just like the screenshot in the description)

## Pre-merge Checklist
